tools/release/version-name/parse-version.ts

```diff
@@ -2,47 +2,49 @@
 const versionNameRegex = /^(\d+)\.(\d+)\.(\d+)(?:-(alpha|beta|rc)\.(\d)+)?$/;
 
 export class Version {
-
   constructor(
-    /** Major version number */
-    public major: number,
-    /** Minor version number */
-    public minor: number,
-    /** Patch version number */
-    public patch: number,
-    /** Pre-release label for the version (e.g. alpha, beta, rc) */
-    public prereleaseLabel?: string,
-    /** Number for the pre-release. There can be multiple pre-releases for a version. */
-    public prereleaseNumber?: number) {}
+      /** Major version number */
+      public major: number,
+      /** Minor version number */
+      public minor: number,
+      /** Patch version number */
+      public patch: number,
+      /** Pre-release label for the version (e.g. alpha, beta, rc) */
+      public prereleaseLabel: string|null,
+      /** Number for the pre-release. There can be multiple pre-releases for a version. */
+      public prereleaseNumber: number|null) {}
 
   /** Serializes the version info into a string formatted version name. */
   format(): string {
     return serializeVersion(this);
   }
 
   clone(): Version {
-    return new Version(this.major, this.minor, this.patch, this.prereleaseLabel,
-      this.prereleaseNumber);
+    return new Version(
+        this.major, this.minor, this.patch, this.prereleaseLabel, this.prereleaseNumber);
+  }
+
+  equalsTo(other: Version): boolean {
```

Comment:
Just `equals`?

Reply:
Oh just saw this. I'm fine going with `equals`. Will update this as part of a follow-up PR that touches the release tools :+1: